entrypoint.sh

```diff
@@ -64,7 +64,14 @@ if [ -z "$GITHUB_REPOSITORY" ]; then
     error "GITHUB_REPOSITORY environment variable is not set"
     exit 1
 fi
-GIT_REPOSITORY_URL="https://${GITHUB_PERSONAL_ACCESS_TOKEN}@github.com/$GITHUB_REPOSITORY.wiki.git"
+
+if [ -z "$GITHUB_SERVER_URL" ]; then
+    GITHUB_SERVER="github.com"
+else 
+    GITHUB_SERVER=${GITHUB_SERVER_URL#"https://"}
+fi
```

Comment:
Or would you prefer in a single line?

```suggestion
GITHUB_SERVER=${${GITHUB_SERVER_URL:=https://github.com}#"https://"}
```

Reply:
Tbh I don't care if it is single line or multi line, that is something for the maintainer's style guide decisions. I think that if you don't introduce a breaking change, your chances of getting a PR accepted are higher :)